src/components/Hero/HubHero/index.tsx

```diff
@@ -37,7 +53,7 @@ const HubHero = (props: CommonHeroProps) => {
         borderRadius={{ xl: "base" }}
         bg={{ xl: "hubHeroContentBg" }}
         position={{ xl: "absolute" }}
-        insetStart={{ xl: "8" }}
+        style={{ insetInlineStart }}
```

Comment:
This change was made throughout as the style props `insetStart/End` or `insetInlineStart/End` have not been working as expected with Chakra components. Using `style` fixes this, but then we lose access to the Chakra tokens. As an alternative to using css `var(--eth-type-token)` syntax, the `useToken` hook was used.